README.md

```diff
@@ -32,6 +32,16 @@ This will create all relevant resources required for the Operator to work. Here
 - `ClusterRoleBinding`: This cluster wide binding binds the service account `minio-operator-sa` to cluster role `minio-operator-role`.
 - `Deployment`: Deployment creates a pod using the MinIO-Operator Docker image. This is where the custom controller runs and looks after any changes in custom resource.
 
+To specify the cluster domain instead of the default `cluster.local`, add an environment variable to the pod spec on the deployment.
+
+```
+env:
+    - name: CLUSTER_DOMAIN
+      value: mycluster.mydomain
+```
+
+Or modify the file `minio-operator-with-cluster-name.yaml`
```

Comment:
Can we add this under the `Environment variables` section

Reply:
Ok, good idea